guides/common/modules/proc_registering-hosts.adoc

```diff
@@ -16,13 +16,19 @@ endif::[]
 * You must have root privileges on the host that you want to register.
 ifdef::satellite,orcharhino[]
 * You must have an activation key created.
+endif::[]
+ifdef::satellite[]
 * Optional: If you want to register hosts to Red{nbsp}Hat Insights, you must synchronize the `{RepoRHEL7Server}` repository and make it available in the activation key that you use.
 This is required to install the `insights-client` package on hosts.
+endif::[]
+ifdef::satellite,orcharhino[]
 * {ProjectServer}, any {SmartProxyServer}s, and all hosts must be synchronized with the same NTP server, and have a time synchronization tool enabled and running.
 * The daemon *rhsmcertd* must be running on the hosts.
 * An activation key must be available for the host.
 For more information, see {ContentManagementDocURL}Managing_Activation_Keys_content-management[Managing Activation Keys] in _{ContentManagementDocTitle}_.
 * Subscription Manager must be version 1.10 or later.
+endif::[]
+ifdef::satellite[]
 The package is available in the standard {RHEL} repository.
 endif::[]
```

Comment:
These don't appear to be ordered so I am for simplifying
```suggestion
* You must have root privileges on the host that you want to register.
ifdef::satellite,orcharhino[]
* You must have an activation key created.
* {ProjectServer}, any {SmartProxyServer}s, and all hosts must be synchronized with the same NTP server, and have a time synchronization tool enabled and running.
* The daemon *rhsmcertd* must be running on the hosts.
* An activation key must be available for the host.
For more information, see {ContentManagementDocURL}Managing_Activation_Keys_content-management[Managing Activation Keys] in _{ContentManagementDocTitle}_.
* Subscription Manager must be version 1.10 or later.
endif::[]
ifdef::satellite[]
The package is available in the standard {RHEL} repository.
* Optional: If you want to register hosts to Red{nbsp}Hat Insights, you must synchronize the `{RepoRHEL7Server}` repository and make it available in the activation key that you use.
This is required to install the `insights-client` package on hosts.
endif::[]
```

Reply:
Applied.